src/libcalamaresui/ViewManager.cpp

```diff
@@ -369,6 +369,7 @@ ViewManager::next()
             UPDATE_BUTTON_PROPERTY( backEnabled, false )
         }
         updateCancelEnabled( !settings->disableCancel() && !( executing && settings->disableCancelDuringExec() ) );
+        updateBackAndNextVisibility(!executing || !settings->hideBackAndNextDuringExec());
```

Comment:
Apply the coding-style tool (it will put in some more spaces).

Reply:
Done